crontab.js

```diff
@@ -99,7 +106,7 @@ exports.set_crontab = function(env_vars, callback){
 				}
 
 				if (tab.mailing && JSON.stringify(tab.mailing) != "{}"){
-					crontab_string += "; /usr/local/bin/node " + __dirname + "/bin/crontab-ui-mailer.js " + tab._id + " " + stdout + " " + stderr;
```

Comment:
Thanks for your contribution. `/usr/local/bin/node` is required as sometimes `env` doesn't tell us where the node is.